tests/e2e/040_bridge_domain_test.go

```diff
@@ -22,10 +22,10 @@ import (
 	. "github.com/onsi/gomega"
 	"github.com/onsi/gomega/types"
 
-	"github.com/ligato/vpp-agent/api/models/linux/interfaces"
-	"github.com/ligato/vpp-agent/api/models/linux/namespace"
-	"github.com/ligato/vpp-agent/api/models/vpp/interfaces"
-	"github.com/ligato/vpp-agent/api/models/vpp/l2"
+	linux_interfaces "github.com/ligato/vpp-agent/api/models/linux/interfaces"
+	linux_namespace "github.com/ligato/vpp-agent/api/models/linux/namespace"
+	vpp_interfaces "github.com/ligato/vpp-agent/api/models/vpp/interfaces"
+	vpp_l2 "github.com/ligato/vpp-agent/api/models/vpp/l2"
 	kvs "github.com/ligato/vpp-agent/plugins/kvscheduler/api"
```

Comment:
Aren't these aliases redundant?

Reply:
one of tools in VSCode insist about this

Reply:
```
➜ goimports -d 040_bridge_domain_test.go
diff -u 040_bridge_domain_test.go.orig 040_bridge_domain_test.go
--- 040_bridge_domain_test.go.orig      2019-10-04 11:39:07.309005126 +0300
+++ 040_bridge_domain_test.go   2019-10-04 11:39:07.309005126 +0300
@@ -22,10 +22,10 @@
        . "github.com/onsi/gomega"
        "github.com/onsi/gomega/types"

-       "github.com/ligato/vpp-agent/api/models/linux/interfaces"
-       "github.com/ligato/vpp-agent/api/models/linux/namespace"
-       "github.com/ligato/vpp-agent/api/models/vpp/interfaces"
-       "github.com/ligato/vpp-agent/api/models/vpp/l2"
+       linux_interfaces "github.com/ligato/vpp-agent/api/models/linux/interfaces"
+       linux_namespace "github.com/ligato/vpp-agent/api/models/linux/namespace"
+       vpp_interfaces "github.com/ligato/vpp-agent/api/models/vpp/interfaces"
+       vpp_l2 "github.com/ligato/vpp-agent/api/models/vpp/l2"
        kvs "github.com/ligato/vpp-agent/plugins/kvscheduler/api"
 )
```

Reply:
I suspect it's because name of directory does not match package name.